packages/calling/src/CallingClient/calling/types.ts

```diff
@@ -194,52 +200,259 @@ export type CallRtpStats = {
+
+  /**
+   * Completes a call transfer.
+   *
+   * @param transferType - The type of transfer to perform. Eg. BLIND or CONSULT.
+   * @param transferCallId - The call identifier for the transfer incase of Consult transfer (optional).
+   * @param transferTarget - The target for the transfer incase of Blind transfer(optional).
+   * @example
+   * ```
+   * // blind transfer
+   * call.completeTransfer('BLIND', undefined, '5998');
+   *
+   * // consult transfer
+   * call.completeTransfer('CONSULT', secondCall.getCallId(), undefined);
+   * ```
+   */
+  completeTransfer(
     transferType: TransferType,
     transferCallId?: CallId,
     transferTarget?: string
-  ) => void;
-  getBroadworksCorrelationInfo: () => string | undefined;
-  setBroadworksCorrelationInfo: (broadworksCorrelationInfo: string) => void;
-  getCallRtpStats: () => Promise<CallRtpStats>;
+  ): void;
+
+  /**
+   * Gets information related to the call's Broadworks correlationId.
+   *
+   * @returns The Broadworks correlation information or undefined if not available.
+   */
+  getBroadworksCorrelationInfo(): string | undefined;
+
+  /**
+   * Sets the Broadworks correlation information for the call.
+   *
+   * @param broadworksCorrelationInfo - The Broadworks correlation information.
+   */
+  setBroadworksCorrelationInfo(broadworksCorrelationInfo: string): void;
+
+  /**
+   * Gets RTP (Real-time Transport Protocol) statistics for the call.
+   *
+   * @returns A promise that resolves to the RTP statistics for the call.
+   */
+  getCallRtpStats(): Promise<CallRtpStats>;
 }
 
 export type DeleteRecordCallBack = (callId: CallId) => void;
 export type CallEmitterCallBack = (callerInfo: DisplayInformation) => void;
 export type CallErrorEmitterCallBack = (error: CallError) => void;
 export type RetryCallBack = (interval: number) => void;
 
+/**
+ * Represents an interface for managing calls within a call manager.
+ */
 export interface ICallManager extends Eventing<CallEventTypes> {
-  createCall: (
+  /**
+   * Creates a call with the specified details.
+   *
+   * @param destination - The call details including destination information.
+   * @param direction - The direction of the call (e.g., incoming or outgoing).
+   * @param deviceId - The unique identifier of the device associated with the call.
+   * @param lineId - The identifier of the line to which the call belongs.
+   * @returns The created call object as {@link ICall}.
+   */
+  createCall(
     destination: CallDetails,
     direction: CallDirection,
     deviceId: string,
     lineId: string
-  ) => ICall;
-  endCall: (callId: CallId) => void;
-  getCall: (callId: CallId) => ICall;
-  updateActiveMobius: (url: string) => void;
-  getActiveCalls: () => Record<string, ICall>;
-  updateLine: (deviceId: string, line: ILine) => void;
+  ): ICall;
+
+  /**
+   * Retrieves a call based on its unique call identifier.
+   *
+   * @param correlationId - The unique identifier for the call at client
```

Comment:
As per our discussion offline, we have decided to remove ICallManager from the documentation. I have already written the docs for this class, this did not remove it as it could be useful for internal reference. Do you still want me to add examples?